### PR TITLE
introduce Server-installation/Overview section

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issue: https://tracker.mender.io/browse/MEN-3220

### CODE REVIEW

File: 06.Server-installation/01.Overview/docs.md [hunk 1]
```diff
@@ -4,3 +4,62 @@ taxonomy:
     category: docs
     label: reference
 ---
+
+Mender backend consits of a number microservices, each implementing a
```
Comment (drewmoseley) [inline 06.Server-installation/01.Overview/docs.md:8]: The Mender backend consists

File: 06.Server-installation/01.Overview/docs.md [hunk 2]
```diff
@@ -4,3 +4,62 @@ taxonomy:
...
+- [Mender Device Authentication Service](https://github.com/mendersoftware/deviceauth?target=_blank)
+- [Mender Deployment Service](https://github.com/mendersoftware/deployments?target=_blank)
+- [Mender Device Inventory Service](https://github.com/mendersoftware/inventory?target=_blank)
+- [Mender User Administration Service](https://github.com/mendersoftware/useradm?target=_blank)
+- [Mender API Gateway](https://github.com/mendersoftware/mender-api-gateway-docker?target=_blank)
+- [Minio](https://www.minio.io/?target=_blank) object storage
+- Storage service proxy based on [OpenResty](https://openresty.org/en/?target=_blank)
+
+Services are delivered in form of Docker images, available from
+official [Mender Docker repository](https://hub.docker.com/r/mendersoftware/?target=_blank).
```
Comment (drewmoseley) [inline 06.Server-installation/01.Overview/docs.md:20]: the official...

File: 06.Server-installation/01.Overview/docs.md (hunk 1 above)
Comment (kjaskiewiczz) [reply 06.Server-installation/01.Overview/docs.md]: fixed

File: 06.Server-installation/01.Overview/docs.md (hunk 2 above)
Comment (kjaskiewiczz) [reply 06.Server-installation/01.Overview/docs.md]: fixed

File: 06.Server-installation/01.Overview/docs.md [hunk 3]
```diff
@@ -4,3 +4,62 @@ taxonomy:
     category: docs
     label: reference
 ---
+
+The Mender backend consits of a number microservices, each implementing a
+small, well defined piece of functionality.
```
Comment (estenberg) [inline 06.Server-installation/01.Overview/docs.md:9]: "piece" sounds a bit informal - perhaps "area of responsibility"?

Comment (kjaskiewiczz) [reply 06.Server-installation/01.Overview/docs.md:9]: according to google, this phrase is commonly used even in the books

**Pull request merged**